parking/backend/user_server/wsserver.py

```diff
@@ -0,0 +1,74 @@
+import logging
+from typing import Dict, List
+
+import attr
+from tornado import websocket
+
+import parking.shared.ws_models as models
+from parking.shared.location import Location
+
+
+class UserWSHandler(websocket.WebSocketHandler):
+    """The WebSocket handler. Handles user ws connections."""
+    def check_origin(self, origin) -> bool:
+        return True
+
+    def initialize(self, sessions_handler: 'SessionsHandler') -> None:
+        self.sh = sessions_handler
+
+    def open(self, user_id: str) -> None:
+        self.user_id = user_id
+        logging.info("WebSocket opened for user_id = '{}'".format(self.user_id))
```

Comment:
Perhaps have a dedicated backend logger?

Like put `logger = logging.getLogger('backend')` at the top rather than calling the root logger directly

Although this is very nit-picky :)